tests/unit/relations/test_db.py

```diff
@@ -26,40 +30,61 @@ def setUp(self):
         self.harness.begin()
 
         self.charm = self.harness.charm
+        self.app = self.charm.app.name
+        self.unit = self.charm.unit.name
+        self.backend = self.charm.backend
         self.db_relation = self.charm.legacy_db_relation
         self.db_admin_relation = self.charm.legacy_db_admin_relation
 
+        # TODO update mocks now we're using the mock harness
```

Comment:
remove? or which mocks are you referring to

Reply:
removed :)